packaging/src/docker/6.x/docker-compose.yml

```diff
@@ -1,51 +1,48 @@
 version: '3'
 services:
   alfresco:
-      image: quay.io/alfresco/alfresco-content-repository${ALFRESCO_EDITION}:${ALFRESCO_TAG}
-      environment:
-          JAVA_OPTS : "
-              -Ddb.driver=org.postgresql.Driver
-              -Ddb.username=alfresco
-              -Ddb.password=alfresco
-              -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
-              -Dsolr.host=search
-              -Dsolr.port=8983
-              -Dsolr.secureComms=none
-              -Dsolr.base.url=/solr
-              -Dindex.subsystem.name=solr6
-              "
-      ports:
-          - "7203:7203" #JMX connect via service:jmx:rmi:///jndi/rmi://localhost:7203/jmxrmi
-          - "5005:5005" #Java debugging
-          - "8081:8080" #Browser port for Alfresco
-
+    image: alfresco/alfresco-content-repository${ALFRESCO_EDITION}:${ALFRESCO_TAG}
+    environment:
+      JAVA_OPTS : "
+        -Ddb.driver=org.postgresql.Driver
+        -Ddb.username=alfresco
+        -Ddb.password=alfresco
+        -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
+        -Dsolr.host=search
+        -Dsolr.port=8983
```

Comment:
Presumably this works (and it's not something you've changed anyway), but I didn't understand why this isn't 8083 given the port mapping for the search container.

Reply:
Tuna explained this to me. The default behaviour for docker-compose is to add all containers to the same network, so they communicate with each other using the internal port numbers. See the docs for more details: https://docs.docker.com/compose/networking/